### ISSUE TITLE
Dows not work when using with Cocoapods

### ISSUE THREAD

**Issue by Urkman** [leoru/SwiftLoader#2]
To reproduce:
1.) Create new xocde project (Single View)
2.) add the following podfile

```
platform :ios, "8.0"
use_frameworks!

pod 'SwiftLoader'
pod 'Alamofire'
```

3.) run `pod install`
4.) open workspace file
5.) edit ViewController.swift
5a) add `import SwiftLoader`
5b) add `SwiftLoader.show(animated: true)` to viewDidLoad()
6.) compile
7.) get this error: `Module 'SwiftLoader' has no member named 'show'`

btw. AlamoFire works fine...

**Comment by Urkman** [comment]
The Problem is, that your class and the needed functions are not public...

**Comment by leoru** [comment]
Hi! Yes, the problem was in my class and methods which was not public.
I fixed this issue in 0.2 version.
